bool detectedBird
float32 bird_height
float32 bird_width